apparmor.d/groups/systemd/systemd-hostnamed

```diff
@@ -8,7 +8,7 @@ abi <abi/3.0>,
 include <tunables/global>
 
 @{exec_path} = /{usr/,}lib/systemd/systemd-hostnamed
-profile systemd-hostnamed @{exec_path} flags=(attach_disconnected) {
+profile systemd-hostnamed @{exec_path}  flags=(attach_disconnected complain) {
```

Comment:
Do add `complain` here. Add it in the `dists/flags/main.flag` file.

Reply:
It's fixed, that's not a final commit.